docs/bigquery-usage.rst

```diff
@@ -315,9 +313,9 @@ Background a query, loading the results into a table:
    """
    >>> dataset = client.dataset('dataset_name')
    >>> table = dataset.table(name='person_ages')
-   >>> job = client.run_async_query(query,
-   ...                              destination=table,
-   ...                              write_disposition='truncate')
+   >>> job = client.run_async_query('fullname-age', query)
+   >>> job.destination_table = table
+   >>> job.write_disposition= 'truncate'
```

Comment:
I didn't look at the source before, I now realize it is the job name. Sorry.

As per my comment about `copy_table`, could you include the word `job` in the string (so it's clear) and then change the positional arg to the method from `name` to `job_name`.

----

RE: 

> so I shouldn't be passing it here

I think you meant elsewhere? This one is `run_async_query`